### ISSUE TITLE
shell app doesn't update screen reader with command output

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#672]
Screen readers don't really read a lot of the output from commands in the shell app.

So if you type, say `ls` it's not likely to say anything at all. Where as if you type `whoami` it may state the command prompt (whatever `$PS1` looks like onthe current line, it doesn't read the previous line - the output of whoami).  The _most_ I got out of it was reading the `$PS1` prompt, it never said the output of a given command I'd ran.

Tested with NVDA + Windows + Chrome.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201737586037557) by [Unito](https://www.unito.io)
